### ISSUE TITLE
AccessKit no longer detects when Orca is running

### ISSUE THREAD

**Issue by RastislavKish** [AccessKit/accesskit#713]
I'm not sure how to properly test AccessKit itself on this, but since a certain point, frameworks using accessKit seem no longer able to detect Orca, and in turn, the a11y support is not turned on. eframe 0.34.1, Slint 1.16 and Vizia 0.4 from what I've tested are all not accessible on Linux currently. The older version of these frameworks, e.g. Slint 1.13 seem to work correctly.

## Environment

openSUSE Tumbleweed from start of this month, Gnome 49 Wayland, Orca 51.alpha (tested with 49.5 as well, same result).

**Comment by DataTriny** [comment]
Likely a result of #628, please confirm by running the following command which should return true if you have turned on Orca in your settings:

```bash
busctl get-property --user org.a11y.Bus /org/a11y/bus org.a11y.Status ScreenReaderEnabled
```

`ScreenReaderEnabled` is not the right property to monitor, but many users reported that AccessKit was activating on their system even though they did not enable any assistive technology.

I investigated a bit and I think I have found one possible cause. The very first time Orca is started the `IsEnabled` property on `/org/a11y/bus` is set to true. However it seems that there is no path to set it back to false: not when Orca shuts down, not when the AT-SPI registry is stopped and not even after a reboot.

Until this bug in the AT-SPI stack is fixed and the change is reverted here, please turn on your assistive technology through GNOME's designed path, or manually flip the `ScreenReaderEnabled` property.

**Comment by RastislavKish** [comment]
@DataTriny you're right, I've run:

```sh
busctl get-property --user org.a11y.Bus /org/a11y/bus org.a11y.Status ScreenReaderEnabled
```

As you've suggested and the value is false. I've flipped it to true and now Egui as well as Slint work as expected. Interestingly, I do have Orca enabled using the ordinary Gnome method i.e. by pressing Alt+Super+S and Gnome settings show Orca as active, so apparently this flag is not operated by either Gnome or Orca itself.

@mgorse you're an expert on this, do you have any thoughts on how to proceed here?

**Comment by DataTriny** [comment]
I have filed [GNOME/at-spi2-core#224](https://gitlab.gnome.org/GNOME/at-spi2-core/-/issues/224) to describe the current situation as I see it. I will keep this issue open for other people that might stumble upon it.
